src/openforms/utils/decorators.py

```diff
@@ -65,3 +67,36 @@ def _wrapped_view_func(request, *args, **kwargs):
         return _wrapped_view_func
 
     return decorator
+
+
+def mixin_exception_handler(
```

Comment:
Move to utils/admin.py

Reply:
I checked that file and it's actually registering models for real, so perhaps we shouldn't pollute it with utilities. Instead, let's put this in `openforms.admin.decorators` :)